### ISSUE TITLE
"pytorch-ignite" are not available from current channels

### ISSUE THREAD

**Issue by GloryyrolG** [y0ast/Glow-PyTorch#18]
Hi contributors,

It seems `pytorch-ignite` cannot be installed by the command

```python
conda install pytorch=1.1.0 torchvision pytorch-ignite tqdm cudatoolkit=10.0 -c pytorch
```

How do you make the repo scripts work? Thanks.

**Comment by GloryyrolG** [comment]
Then I solved with pytorch=1.3.0 and conda install ignite since the minimal pytorch dependency for ignite is 1.3.0.

**Comment by y0ast** [comment]
newer versions of pytorch should be fine :) !
